packages/alea-frontend/pages/blog/[postId].tsx

```diff
@@ -53,6 +48,15 @@ const BlogPostPage: NextPage = ({ post }: { post: BlogPost }) => {
     }
   }, [router.isReady, postId, router]);
 
+  useEffect(() => {
+    async function checkIsUserCanDeleteOrEdit() {
+      if(await canAccessResource('/blog', Action.UPDATE)) {
```

Comment:
Don't use `/blog` directly here. Move `blogResourceId` to `utils/src/lib/resource-action-utils.ts` so that it can be used by the frontend also.